File: solutions/typescript/VB_T2_014_combined_length.ts
// VB-T2-014: Combined String Length -- TypeScript baseline
function combinedLength(a: string, b: string): number { return a.length + b.length; }
console.assert(combinedLength("hello", "world") === 10);
console.assert(combinedLength("", "") === 0);
console.assert(combinedLength("a", "bc") === 3);
console.assert(combinedLength("test", "") === 4);

File: solutions/python/VB_T2_014_combined_length.py
"""VB-T2-014: Combined String Length -- Python baseline."""
def combined_length(a: str, b: str) -> int:
    return len(a) + len(b)

if __name__ == "__main__":
    assert combined_length("hello", "world") == 10
    assert combined_length("", "") == 0
    assert combined_length("a", "bc") == 3
    assert combined_length("test", "") == 4
    print("VB-T2-014 ok")
